sig-and-wg/sig-core/README.md

```diff
@@ -15,35 +15,18 @@ The SIG Core:
 decision-making process
 * Represents its members and the Kyma community in the Council meetings
 
-## Attendees
-
-These are the mandatory and optional attendees of the SIG Core meetings:
-*	Any member of the Kyma organization (optional)
-*	Any person from outside of the organization (optional)
-* Kyma Product Owners (at least one representative is mandatory)
-* Kyma Scrum Masters (at least one representative is mandatory)
-* SIG Community representative (leader)
-* SDMs (optional)
-* Release manager (optional)
-* Program manager (optional)
-
-SIG Core members nominate leaders who organize, coordinate, and moderate the SIG meetings and discussions. They are also the first point of contact for the Kyma SIG Core members. In its initial stage, the SIG Core leaders are the members of the Kyma organization.
-
 ## Contact
 
 * [Slack](https://kyma-community.slack.com/messages/CBP7LKRPS)
-
-> **NOTE:** This information is not yet available.
-
-* [Open community issues and pull requests](https://github.com/kyma-project/community/labels/{label-name})
+* [Open community issues and pull requests](https://github.com/kyma-project/community/labels/sig%2Fcore)
 
 ## Meetings
 
 > **NOTE:**  Time and frequency is not yet available.
 
-* [Slack](https://kyma-community.slack.com/messages/CBP7LKRPS)
+* [Zoom](https://zoom.us/j/346707320)
 * Frequency: {daily | weekly | bi-weekly | monthly}
```

Comment:
I think we should add change this to `To be decided on first meeting` to not confuse the reader.